config/update_config.bash

```diff
@@ -29,6 +29,27 @@ if [ 0 -eq "${sp_installed}" ]; then
   sudo systemctl disable shairport-sync.service
 fi
 
+# rough configuration of shairport-sync-metadata-reader
+git_installed=$(sudo apt list --installed 2> /dev/null | grep git/ -c)
```

Comment:
why is there a '/' here?

Reply:
This should also check for the package **build-essential** that checks for make and gcc and it looks like it also needs the package **autoconf**

Reply:
Searching for 'git/' actually provides the info that we want - searching for 'git' without the slash ends up showing nine results vs. the one for git/. I'll add checks for build-essential and autoconf

Reply:
Should stm32flash be added to update_config.bash? It isn't necessary for normal operation, but it is required for flashing the preamp firmware.